project/Build.scala

```diff
@@ -1419,4 +1424,85 @@ object Build extends sbt.Build {
           }
       )
   ).dependsOn(partest % "test", library)
+
+  lazy val scalaTestSuite: Project = Project(
+    id = "scalaTestSuite",
+    base = file("scala-test-suite"),
+    settings = commonSettings ++ myScalaJSSettings ++ Seq(
+      publishArtifact in Compile := false,
+
+      testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a"),
+
+      unmanagedSources in Test ++= {
+        def loadList(listName: String): Set[String] = {
+          val listsDir = (resourceDirectory in Test).value / scalaVersion.value
+          val buff = scala.io.Source.fromFile(listsDir / listName)
```

Comment:
It fails with an exception that says that the file does not exist. Considering that it also fails if the file contains incoherent contents, this seams like the equivalent behaviour.